src/cli/commands/add.js

```diff
@@ -70,6 +108,7 @@ export class Add extends Install {
    */
 
   async init(): Promise<Array<string>> {
+    this.addedPatterns = [];
```

Comment:
Okay, sounds good. It was just at a first glance without testing it so I wasn't sure.